src/Resources/app/administration/package.json

```diff
@@ -7,7 +7,8 @@
   "license": "MIT",
   "scripts": {
     "lint": "eslint --ext .js,.ts,.vue,.html,.html.twig .",
-    "lint-fix": "npm run lint -- --fix"
+    "lint-fix": "npm run lint -- --fix",
+    "lint:css": "stylelint ./**/*.scss"
```

Comment:
Also I just took a look at platform admin:
https://github.com/shopware/shopware/blob/207f83b14ef269b6e8186b6e6beb5fa76c3cbb21/src/Administration/Resources/app/administration/package.json#L34-L35

I think it would be great to stay consistent and introduce `lint:scss` + `lint:scss-fix` instead of `lint:css`

Reply:
I updated the command to `lint:scss` and added a fix command for the linting.